packages/auto-approve/src/check-config.ts

```diff
@@ -66,15 +67,81 @@ export function validateYaml(configYaml: string): string {
  */
 export async function validateSchema(
   configYaml: string | object
-): Promise<ErrorMessage[] | undefined> {
+): Promise<string> {
   const parsedYaml =
     typeof configYaml === 'string' ? yaml.load(configYaml) : configYaml;
   const validateSchema = await ajv.compile(schema);
   await validateSchema(parsedYaml);
   const errorText = (await validateSchema).errors?.map(x => {
     return {wrongProperty: x.params, message: x.message};
   });
-  return errorText;
+  return JSON.stringify(errorText) ?? '';
+}
+
+/**
+ * Checks the location of auto-approve, whether it's on the PR or branch, and performs checks if it exists
+ *
+ * @param octokit Octokit instance to make calls to github API
+ * @param owner of the repo of the incoming PR
+ * @param repo of the incoming PR
+ * @param autoApproveFile if the incoming PR includes an auto-approve file, that file; undefined if not
+ * @returns empty string if valid, otherwise an error message.
+ */
+export async function checkAutoApprove(
```

Comment:
I would be tempted to call this `checkAutoApproveConfig`, if I'm understanding the method properly.